caffe2/perfkernels/embedding_lookup_idx.cc

```diff
@@ -185,10 +185,8 @@ static bool EmbeddingLookupGenericSlowIdx(
       return;                                                                                         \
     }                                                                                                 \
     int64_t current = 0;                                                                              \
```

Comment:
Without the suggested changes on the caller side, this doesn't match with what EmbeddingLookupGenericSlowIdx is doing